### PR TITLE
Feature/214 side menu on members page

### LINKED ISSUE #214: Side menu on members page

### PR DESCRIPTION
**Pull request by n-s-mordvinov**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -168,9 +171,11 @@
     "nameDesc": "Name Z to A"
   },
   "memberItemActionBlock": {
-    "editName": "Edit name",
+    "deactivate": "Deactivate",
+    "delete": "Delete",
+    "hide": "Hide",
     "editPosition": "Edit position",
-    "deactivated": "Deactivate"
+    "transferTheMemberTo": "Transfer the member to"
```
Comment (kirillta) [inline src/locales/en/translation.json:178]: `Transfer member` is better

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -168,9 +171,11 @@
     "nameDesc": "Name Z to A"
   },
   "memberItemActionBlock": {
-    "editName": "Edit name",
+    "deactivate": "Deactivate",
+    "delete": "Delete",
```
Comment (kirillta) [inline src/locales/en/translation.json:175]: `Remove`

Comment (sernik73) [reply src/locales/en/translation.json]: Done

File: src/locales/en/translation.json (hunk 1 above)
Comment (sernik73) [reply src/locales/en/translation.json]: Done

**Review by kirillta — APPROVED** [review]

**Pull request merged**